src/test/javascript/spec/component/overview/course-lectures/course-lecture-details.component.spec.ts

```diff
@@ -173,6 +174,31 @@ describe('CourseLectureDetails', () => {
         expect(courseLecturesDetailsComponent.hasPdfLectureUnit).toBeFalse();
     }));
 
+    it('should display manage button', fakeAsync(() => {
+        lecture.course!.isAtLeastTutor = true;
+        fixture.detectChanges();
+
+        const manageLectureButton = debugElement.query(By.css('#manageLectureButton'));
+        expect(manageLectureButton).not.toBeNull();
+    }));
+
+    it('should not display manage button', fakeAsync(() => {
+        lecture.course!.isAtLeastTutor = false;
+        fixture.detectChanges();
+
+        const manageLectureButton = debugElement.query(By.css('#manageLectureButton'));
+        expect(manageLectureButton).toBeNull();
+    }));
```

Comment:
(optional) Basically the same as in the other test above.

(required) If you don't combine the tests to a parameterized test, please clarify the test case descriptions. Having "it should show the button" and "it should not show the button" is kinda contradicting each other without further clarification on the condition, e.g. "show if user is at least tutor"

Reply:
I've changed names of those tests